crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -269,6 +272,8 @@ impl AcirContext {
                 let data_expr = data.to_expression();
                 let witness = self.acir_ir.get_or_create_witness(&data_expr);
                 self.acir_ir.range_constraint(witness, *bit_size)?;
+                // Log the bit size for this variable
```

Comment:
Currently, we are only logging the bit size after a range constraint but we should log it when we add a constant because the bit size is known and after certain operations in GeneratedACIR, for example, when we call `more_than_eq` from GeneratedAcir, we know that the result is a boolean

Reply:
Needs an issue

Reply:
https://github.com/noir-lang/noir/issues/1439